game_starter.py

```diff
@@ -67,8 +79,39 @@ def get_guessed_word(secret_word, letters_guessed):
     returns: string, comprised of letters and underscores that represents
       what letters in secret_word have been guessed so far.
     '''
-    # FILL IN YOUR CODE HERE...
-    pass
+    
+    '''
+    output_string = ''
+    for every letter in secret_word
+      if the letter is in letters_guessed
+        concatenate that letter on to output_string
+      otherwise
+        concatenate underscore space on to output_string
+
+      return output_string
+    '''
+
+    '''
+    output_list = []
+    for every letter in secret_word
+      if the letter is in letters_guessed
+        append that letter to output_list
+      otherwise
+        append underscore space
+
+    convery list toa string
+    join all the characters in output_list
+    and return that string
+    '''
+    
+    output_string = ''
+
+    for i in secret_word:
+      if i in letters_guessed:
+        output_string += i
+      else:
+        output_string += '_'
```

Comment:
When inserting underscores into your string, it's a good idea to add at least a space after each 
one, so it's clear to the user how many unguessed letters are left in the string (compare the 
readability of 
____
with 
_ _ _ _ 
).  @Kanyanat-Grace